From 2739ea80b7137daf3f9a39fd3a2ddfd787a6b964 Mon Sep 17 00:00:00 2001
From: James Ross-Gowan <rossymiles@gmail.com>
Date: Wed, 24 Jun 2015 20:49:59 +1000
Subject: [PATCH] Revert "win32: fix desktop directory"

This reverts commit f56a7af9f696aab7f4095f282f22dcbe7c758c9e.

This was incorrectly merged. There's no folder parameter in the version
of the function on the release branch.
---
 osdep/path-win.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/osdep/path-win.c b/osdep/path-win.c
index f22b8a3..26f7ffa 100644
--- a/osdep/path-win.c
+++ b/osdep/path-win.c
@@ -51,7 +51,7 @@ static char *mp_get_win_app_dir(void *talloc_ctx)
 {
     wchar_t w_appdir[MAX_PATH + 1] = {0};
 
-    if (SHGetFolderPathW(NULL, folder|CSIDL_FLAG_CREATE, NULL,
+    if (SHGetFolderPathW(NULL, CSIDL_APPDATA|CSIDL_FLAG_CREATE, NULL,
         SHGFP_TYPE_CURRENT, w_appdir) != S_OK)
         return NULL;
 
-- 
2.4.4

